### PR TITLE
Nitsche fix calculate force

### PR DESCRIPTION
**Pull request by jeannejoachim**
# Description of the problem

Torque and force calculation could not be enabled for independently for each Nitsche solids. This was problematic on multiple solid cases.

# Description of the solution

Torque and force boolean and filename are now defined for each solid.

Other small fixes:
- the file for the analytical solution error is now saved in the output folder
- remove parameter redundancy for the center_of_rotation of Nitsche solids (only parameter parsed now)

# How Has This Been Tested?

Tested on an example with 2 solids (two-bar-mixer) + Nitsche tests were updated with the new parameters.

# Documentation

- Change page for Nitsche prm section

- Change page of Nitsche example + prm in example folder

# Future changes

For now the output table repeats the column names for each solids, so I had to change one of the tests output to take that into account. I think that to change that we should add a new function to generate the table, maybe by parsing it a ghost text used only to fix the column width? Anyway this is really of second order for now.

### CODE REVIEW

File: include/core/boundary_conditions.h [hunk 1]
```diff
@@ -678,6 +681,162 @@ namespace BoundaryConditions
...
+  /**
+   * @brief This class manages the boundary conditions for VOF solver
+   * It introduces the boundary functions and declares the boundary conditions
+   * coherently. (TODO update)
+   * The members ... contain double used for bc calculation :
+   *  - if bc type is "..." (... condition), "..." is the
+   * double passed to the deal.ii ConstantFunction
+   */
+
+  template <int dim>
```
Comment (blaisb) [inline include/core/boundary_conditions.h:693]: I don't understand how this is related to this PR?

**Review by blaisb — COMMENTED** [review]
Everything seems ok, but you seem to have added some VOF boundary things here?

File: include/core/boundary_conditions.h (hunk 1 above)
Comment (jeannejoachim) [reply include/core/boundary_conditions.h:693]: Hum strange, I worked on separated folders for both. Maybe I got mixed up, I will check that right away!

**Review by oguevremont — APPROVED** [review]
Very clean! I have two esthetic comments, but that is all

File: include/core/nitsche.h [hunk 2]
```diff
@@ -58,6 +58,12 @@ namespace Parameters
     // Information for force calculation
     Point<dim>
       center_of_rotation; // Center of rotation used for torque calculation
+
+    bool        calculate_force_on_solid;
+    bool        calculate_torque_on_solid;
```
[the remark this answers is not in the document]
Comment (jeannejoachim) [reply include/core/nitsche.h:63]: agreed! This bugged me ahah. 
@blaisb if you approve also I can change that

[the remark this answers is not in the document]
Comment (blaisb) [reply include/core/nitsche.h:63]: you can change it, no biggie.

**Review by blaisb — APPROVED** [review]
Good to go on my side

**Review by blaisb — APPROVED** [review]

**Pull request merged**